apps/server/src/services/ElasticSearchClient.ts

```diff
@@ -267,12 +291,12 @@ function buildMustNotQuery(hiddenInstitutions: string[]): any[] {
 }
 
 export async function getInstitution(id: string): Promise<CachedInstitution> {
-  const institutionResponse = await ElasticsearchClient.get({
+  const { body } = await ElasticsearchClient.get({
     id,
     index: "institutions",
   });
 
-  return institutionResponse._source as CachedInstitution;
+  return body._source as CachedInstitution;
```

Comment:
Is this change because of openSource?

Reply:
Yeah it's the difference between Elasticsearch version 8 and 7.10